src/main/resources/config/liquibase/subject.csv

```diff
@@ -1,52 +1,52 @@
-"ID";"EXTERNAL_LINK";"EXTERNAL_ID";"REMOVED";"USER_ID";"created_by"
```

Comment:
Similar to above
> No changes should be made to existing Liquibase changeSets, because it leads to undefined state of the database. If you want to add a default value (I think we shouldn't, it should be null), a new changeset should be created.